### PR TITLE
data_upload

### PR DESCRIPTION
**Pull request by rafecchang**

### CODE REVIEW

File: data/.DS_Store
Comment (Marcony1) [inline data/.DS_Store:1]: I think this file should not be downloaded. Could you add it to the gitignore, please? Thanks

File: data/raw/.DS_Store
Comment (Marcony1) [inline data/raw/.DS_Store:1]: I think this file should not be downloaded. Could you add it to the gitignore, please? Thanks

**Review by Marcony1 — CHANGES_REQUESTED** [review]

File: data/.DS_Store
Comment (rafecchang) [reply data/.DS_Store:1]: Why is that? How are is the dashboard going to calculate how much money users save without using the electricity price?

Comment (Marcony1) [reply data/.DS_Store:1]: Oh, I am talking about the .DS_Store file. Apparently it was pushed as well

Comment (rafecchang) [reply data/.DS_Store:1]: I see. I uploaded the file to keep the structure of data/raw/. I am not sure if that ca be retained if I only upload the csv. We can merge then I will create another pull req to delete the .DS_store files.

**Review by Marcony1 — APPROVED** [review]

**Pull request merged**